.nvmrc

```diff
@@ -1 +1 @@
-lts/iron
+22.11.0
```

Comment:
Can someone confirm if `nvm use` works correctly with `lts/jod`?

I've temporarily used ~`lts/jod`~ `22.11.0` until this PR is merged to enable `lts/jod` in [asdf](https://asdf-vm.com)

* https://github.com/asdf-vm/asdf-nodejs/pull/404

If it's helpful, I've got some old setup notes in https://github.com/alphagov/govuk-frontend/pull/4201#issuecomment-1784817878